notify/impl.go

```diff
@@ -477,11 +477,13 @@ func (n *PagerDuty) notifyV1(
 		Details:     details,
 	}
 
-	apiURL, err := url.Parse("https://events.pagerduty.com/generic/2010-04-15/create_event.json")
-	if err != nil {
-		return false, err
+	if n.conf.URL.String() == "https://events.pagerduty.com/v2/enqueue" {
```

Comment:
> .com/**v2**/enqueue

Why are we talking to the `v2` api in the `notifyV1` function?

Reply:
The `https://events.pagerduty.com/v2/enqueue` value is the default value in alertmanager for the v2 api. The proposed logic checks whether that is the configured url (which would mean it hasn't been overridden) before switching to the default v1 api url. This allows the url to be overridden and used for the v1 api.